pyat/at/physics/radiation.py

```diff
@@ -31,36 +31,39 @@ def ohmi_envelope(ring, refpts=None, orbit=None, keep_lattice=False):
     emit0, mode_emit, damping_rates, tunes, emit = ohmi_envelope(ring[, refpts])
 
     PARAMETERS
-        ring                at.Lattice object
-        refpts          elements at which data is returned. It can be:
+        ring            at.Lattice object
+        refpts=None     elements at which data is returned. It can be:
                         1) an integer in the range [-len(ring), len(ring)-1]
                            selecting the element according to python indexing
                            rules. As a special case, len(ring) is allowed and
                            refers to the end of the last element,
                         2) an ordered list of such integers without duplicates,
                         3) a numpy array of booleans of maximum length
                            len(ring)+1, where selected elements are True.
-                        Defaults to None
 
     KEYWORDS
-        orbit=None          Avoids looking for the colsed orbit if is already known ((6,) array)
-        keep_lattice=False  Assume no lattice change since the previous tracking.
-                            Defaults to False
+        orbit=None          Avoids looking for the closed orbit if it is
+                            already known                           (6,) array)
+        keep_lattice=False  Assume no lattice change since the previous tracking
 
     OUTPUT
         emit0               emittance data at the start/end of the ring
         beamdata            beam parameters at the start of the ring
         emit                emittance data at the points refered to by refpts,
                             if refpts is None an empty structure is returned.
 
-        emit is a structured array with fields:
-        R66                 (6, 6) equilibrium envelope matrix R
-        R44                 (4, 4) betatron emittance matrix (dpp = 0)
-        T66                 (6, 6) transfer matrix from the start of the ring
+        emit is a record array with fields:
+        r66                 (6, 6) equilibrium envelope matrix R
+        r44                 (4, 4) betatron emittance matrix (dpp = 0)
+        m66                 (6, 6) transfer matrix from the start of the ring
         orbit6              (6,) closed orbit
         emitXY              betatron emittance projected on xxp and yyp
         emitXYZ             6x6 emittance projected on xxp, yyp, ldp
```

Comment:
Specify sizes for `emitXY` and `emitXYZ`, I think this is especially important for `emitXYZ` as otherwise the "6x6" may cause confusion.
```
emitXY               (2,) betatron emittance projected on xxp and yyp
emitXYZ             (3,) 6x6 emittance projected on xxp, yyp, ldp
```

Reply:
Good remark!